src/manifests/utils/configs.ts

```diff
@@ -181,7 +181,9 @@ export const corruptorConfigUtils = function (
           // If bitrate is set, filter out segments that doesn't match
           params = params.filter(
             (config) =>
-              !config?.br || config?.br === '*' || config?.br === segmentBitrate
+              !config?.br ||
+              config?.br === '*' ||
+              config?.br.toString().includes(segmentBitrate.toString()) //Checks if .br contains bitrates that need to be filtered
```

Comment:
This part doesn't look safe to me. At this point `params` is JSON parsed, so it should be either a number, the string "`*`" or an array of numbers. We already handled if it's "`*`", so what we need to handle is if it's a number or array of numbers.

I would do something like this: Array.isArray(config?.br) ? config.br : [config.br]. Then you know it's an array and can check .includes consistently. (`Array.from(something)` does something similar, but it's not safe to use in this specific case because it doesn't work with numbers).

I would also break up the filter function a bit more. Since it grew in complexity I think it's better / more clear to add a scope (`{}`) and explicit returns now.

The way you're doing it now will detect false positives, for example if you specify `500000` it will also match with `1500000` or `5000000`